Source/DafnyLanguageServer/Workspace/ITextDocumentLoader.cs

```diff
@@ -26,11 +26,13 @@ public interface ITextDocumentLoader {
     /// Loads the specified document item of the language server and applies the necessary steps
     /// to generate a dafny document instance.
     /// </summary>
+    /// <param name="options"></param>
```

Comment:
I'm generally against these rote documentation elements, even when they do have text if the text just describes the name of the element. Is our build process forcing them?